users/views.py

```diff
@@ -1,3 +1,17 @@
 from django.shortcuts import render
+from django.http import HttpResponse
 
-# Create your views here.
+from users.forms import RegistrationForm
+
+
+def register(request):
+    if request.method == 'POST':
+        form = RegistrationForm(request.POST)
+        if form.is_valid():
+            form.save()
+
+            return HttpResponse("You successfully created a new account!")
+    else:
+        form = RegistrationForm()
+
+    return render(request, 'registration.html', {'form': form})
```

Comment:
Django views have `post` and `get` methods, among many other, that would allow you to omit the if-else branching within the registration view method. Those tend to get messy and hard to debug if the code becomes too large.